lib/minitest/mock.rb

```diff
@@ -68,15 +68,15 @@ def initialize(delegator = nil) # :nodoc:
     #   @mock.uses_one_string("bar") # => true
     #   @mock.verify  # => raises MockExpectationError
 
-    def expect(name, retval, args = [], &blk)
+    def expect(name, retval, args = [], times: nil, &blk)
```

Comment:
Right. Remembered that on the hashes, totally forgot about it here. Thanks for pointing it out.

I think the most important is the interface for the user, so I'd definitely go with an options hash.